### ISSUE TITLE
[Feature] [Accessibility] -  No possibility to read / select content on components

### ISSUE THREAD

**Issue by sLorenzini** [PrestaShopCorp/puik#346]
# What problem does this feature solve?

To improve accessibility on components, some tests are made on several components.
Content of : 
- accordion and accordiongroup 
- alert
- button (disabled)
- badge
- card
- modal
- pagination
- radio (disabled component)
- select (disabled component + default placeholder + searchbar no possibility to select and write on it)
- table
- tooltip content

components coudn't be read by a voiceover tool or selected by using keyboard.

Blind or sand-blind people should have the possibility to ear this content to understand the page.
Invalid people should have the possibility to select this content by using only the keyboard.

Examples :

Accordion / AccordionGroup:
![accordion-accordiongroup-content-selected](https://github.com/PrestaShopCorp/puik/assets/2854834/769218af-a8d5-4610-acde-e97dc619a138)
![accordion-accordiongroup-content-selected-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/040d099c-c644-4851-be9e-90f1e969e7ec)

Alert: 
![alert-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/8c245b25-9298-4196-bd8c-124ed9f59c55)

Card:
![card-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/187209ab-e23d-47c0-9ec2-50dc0b6cf2eb)

Modal:
![modal-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/dba0e445-5328-4e53-9ef1-a3ca1e5d0650)

Pagination:
![pagination-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/c2ead28a-07a1-43f2-9db9-32e804b52fec)

Radio:
![radio-voiceover](https://github.com/PrestaShopCorp/puik/assets/2854834/c2e9ca6e-7bcc-4533-bac1-dcd4fd9e11cb)

Tooltip:
![Capture d’écran 2024-07-18 à 10 53 48](https://github.com/user-attachments/assets/213bf675-7a75-4076-a0c2-58e216c4190f)

# What is your idea for solve this issue

Find a way to allow tools or keyboard selection to read the content added.
To make this test, you can use voiceover software native in mac OS or https://www.nvda-fr.org/  for windows.
